packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -345,11 +350,17 @@ class _HeroState extends State<Hero> {
     });
   }
 
-  void endFlight() {
-    if (mounted) {
-      setState(() {
-        _placeholderSize = null;
-      });
+  void ensurePlaceholderIsHidden() {
+    setState(() {
```

Comment:
This should probably be guarded by `if (mounted)` and the if mounted should be removed from endFlight?